### DISCUSSION TITLE
GitHub Dashboard across repos - Security alerts, GitHub Actions workflow status etc

### DISCUSSION THREAD

**Discussion by HariSekhon** [community/community#15789] [Code Security]
Feature Request to have a unified dashboard for GitHub, or more than 1 dashboard, to address an overview across repos for the following:

- Security Alerts from each or configurable list of repos
- GitHub Actions workflow statuses from each or configurable list of repo

**Comment by jenschelkopf** [comment]
Hi @HariSekhon, I'm part of the code security team at GitHub. Can you tell me more about how you would use this dashboard? Are you interested in this for open source projects, work projects, or a combination? Are you an org admin or a security team member? Do you work in a large organization?

**Reply by HariSekhon** [reply]
I am working for a large organization at the moment, and have seen the Security Overview dashboard for our organization, which is useful for the security alerts part.

A counterpart for the CI/CD statuses would be useful.

I am also a heavy open source user and would like something similar for a personal level too without having to use an organization.

I think this is something that many people would benefit from being able to get a good overview of their repos.

**Reply by rupe120** [reply]
Agreed, I'm looking for this as well for an Enterprise account I'm working with. Having an overview of the status of deployments and other actions over the last 24 hours would be great. 

My context is largely private repositories at the moment, though I would love to see this accessible regardless of the level. I find myself working across many organizations and would love to have a single dashboard of things I care about.

**Reply by jenschelkopf** [reply]
Okay, this is super helpful feedback. I will keep it in mind. Thank you!
